pyproject.toml

```diff
@@ -36,23 +36,24 @@ extend-exclude = [
 
 [tool.ruff.lint]
 extend-select = [
+    "ERA", # eradicate
     "B",   # flake8-bugbear
     "C4",  # flake8-comprehensions
-    "ERA", # flake8-eradicate/eradicate
-    "I",   # isort
-    "N",   # pep8-naming
+    "ISC", # flake8-implicit-str-concat
+    "PTH", # flake8-use-pathlib
     "PIE", # flake8-pie
-    "PGH", # pygrep
-    "RUF", # ruff checks
-    "SIM", # flake8-simplify
     "T20", # flake8-print
-    "TCH", # flake8-type-checking
+    "SIM", # flake8-simplify
     "TID", # flake8-tidy-imports
-    "UP",  # pyupgrade
-    "D",   # pydocstyle
-    "PTH", # flake8-use-pathlib
-    "ISC", # implicit-str-concat
+    "TC",  # flake8-type-checking
+    "FLY", # flynt
```

Comment:
We didn't have this ruleset chosen before. Why did you add it?

Reply:
I find f-strings more readable, but I can remove it.

Reply:
I don't mind having it added, it's just more than expected from the PR title.

Reply:
I'll remove it from this PR.